docs/android/README.md

```diff
@@ -1,4 +1,4 @@
-# Cordova Wrapper for PSPDFKit 5 for Android
+# Cordova Pluging - PSPDFKit for Android v5.5.1
```

Comment:
Are we still supporting v5? Is there any reason why we aren't moving forward with v6?

Reply:
I'd like to make `5.5.1` stable, create a git tag for it (so repo users can go back to that if needed), and then proceed to move to `6.0` right after this. Basically a continuation of what was done before in the Android repo: https://github.com/PSPDFKit/Cordova-Android/releases. I'd like to sync up tomorrow on how we'll handle these since it's monorepo now.